handbook/company/communications.md

```diff
@@ -432,6 +432,19 @@ Estimation points represent the effort required to complete a task. After access
 
 > Larger projects are estimated in a way that can sometimes look disproportionate to account for edge cases that weren't caught during planning. This helps us develop [iteratively](https://fleetdm.com/handbook/company#results) and deliver bite-sized functionality on more predictable time scales.
 
+### T-shirt sizes
+
+T-shirt sizes represent a rough estimate on the effort required to complete a task. T-shirt sizes are used to understand the level-of-effort before the task has gone through [drafting](https://fleetdm.com/handbook/product-design#drafting).
+
+[Estimation points](#estimation-points) are used if a task has already gone through drafting.
+
+| T-shirt size | Time               |
+|:---|:-----------------------------|
+| XS | ≤1 week for 1 contributor     |
+| S  | 1 sprint for 1 contributor   |
+| M  | 1 sprint for 2 contributors  |
+| L  | 1 sprint for the whole team  |
+| XL | ≥1 sprint for the whole team |
```

Comment:
/cc @georgekarrv on this as IIRC this was phrased differently in the doc where these were first brought up. Otherwise it looks like there's more overlap between L and XL than there actually is.

Probably also need to specify point ranges here, since sometimes team capacity varies sprint-to-sprint, and we also have different-sized teams so without points range calibration L/XL will mean quite different things between e.g. Software and Orchestration teams.

Reply:
> Probably also need to specify point ranges here, since sometimes team capacity varies sprint-to-sprint, and we also have different-sized teams

I think it's ok that the teams are different sizes and I'm not sure we need to tie it to points. 

The reason to T-shirt size is for a specific product group to get a rough idea on level of effort for a specific story before drafting. That way, we can know if we're bringing to many stories into a design sprint.

Even if the teams are different in size, I think it's important that each product group is using the same definition for each size so folks outside of the product group can get a rough understanding of level-of-effort.

I'll leave it up to @lukeheath to finalize and merge this in.